var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -25,15 +25,16 @@
 from spack import *
 
 
-class LibjpegTurbo(AutotoolsPackage):
+class LibjpegTurbo(Package):
     """libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to
        accelerate baseline JPEG compression and decompression. libjpeg is a
        library that implements JPEG image encoding, decoding and
        transcoding."""
+    # https://github.com/libjpeg-turbo/libjpeg-turbo/blob/master/BUILDING.md
+    homepage = "https://github.com/libjpeg-turbo/libjpeg-turbo"
```

Comment:
I would actually use https://libjpeg-turbo.org/ as the homepage. That's what the previous homepage redirects to.

Reply:
For some reason the building instructions for the 1.6.x was broken from there but that's fine.